public/js/app/components/AnalysisTable.jsx

```diff
@@ -31,23 +30,24 @@ const ActiveOrDisabledLink = ({ fileName, linkTitle, storageUri, roundName, vali
 
 // NOTE: uses state from & is hard coded to archive page.
 // So less a 'component' and more a part of a page.
-const AnalysisTable = () => {
+const AnalysisTable = ({ translation }) => {
   const [context] = useWebContext()
   const { userLang, courseCode, analysisData } = context
   const { storageUri } = context.browserConfig
-  const translations = i18n.message('archiveTitles', userLang)
 
   const yearsDescending = Object.keys(analysisData).reverse()
   const analysisPerSemester = yearsDescending.flatMap((year) => analysisData[year])
   const analysisDataRows = analysisPerSemester.map((semesterData) => {
```

Comment:
If I'm not mistaken, in the other `Tables`, we have a helper function creating the rows. What do you think of adding one here as well to have the same logic?